contracts/eosio.system/src/wps.cpp

```diff
@@ -161,7 +161,7 @@ namespace eosiosystem {
         eosio::action(
                 eosio::permission_level{get_self() , "active"_n },
```

Comment:
to make the perms work you can use: `eosio::permission_level{saving_account, active_permission },`